src/SwedbankPay.Episerver.Checkout/ISwedbankPayCheckoutService.cs

```diff
@@ -15,10 +15,10 @@ public interface ISwedbankPayCheckoutService : ISwedbankPayService
     {
         Consumer InitiateConsumerSession(CultureInfo currentLanguage, string email = null, string mobilePhone = null, string ssn = null);
         PaymentOrder CreateOrUpdatePaymentOrder(IOrderGroup orderGroup, string description, string consumerProfileRef = null);
-        CheckoutConfiguration LoadCheckoutConfiguration(IMarket market);
+        CheckoutConfiguration LoadCheckoutConfiguration(IMarket market, string languageId);
 
         PaymentOrder GetPaymentOrder(IOrderGroup orderGroup, PaymentOrderExpand paymentOrderExpand = PaymentOrderExpand.None);
-        PaymentOrder GetPaymentOrder(Uri id, IMarket market, PaymentOrderExpand paymentOrderExpand = PaymentOrderExpand.None);
+        PaymentOrder GetPaymentOrder(Uri id, IMarket market, string languageId, PaymentOrderExpand paymentOrderExpand = PaymentOrderExpand.None);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-episerver-checkout/pullRequest?prid=5354564)